# KeypointsWithScores.msg
Header header
geometry_msgs/Point[] keypoints
std_msgs/Float32[] keypoint_scores